execution/specExecutor.go

```diff
@@ -244,7 +244,10 @@ func (e *specExecutor) skipSpecForError(err error) {
 func (e *specExecutor) accumulateSkippedScenarioResults() []result.Result {
 	var scenarioResults []result.Result
 	for _, scenario := range e.specification.Scenarios {
-		scenarioResults = append(scenarioResults, e.getSkippedScenarioResult(scenario))
+		res := e.getSkippedScenarioResult(scenario)
+		event.Notify(event.NewExecutionEvent(event.ScenarioStart, scenario, res, e.stream, *e.currentExecutionInfo))
```

Comment:
Why are `ScenarioStart` and `ScenarioEnd` broadcasted here instead of `executeScenario`?